packages/daimo-api/src/contract/accountFactory.ts

```diff
@@ -0,0 +1,65 @@
+import { accountFactoryConfig } from "@daimo/contract";
+import { Hex, TransactionReceipt } from "viem";
+
+import { ViemClient } from "../chain";
+
+/* Interface to the AccountFactory contract. Creates Daimo accounts. */
+export class AccountFactory {
```

Comment:
i think we're not using this file anywhere

Reply:
the new version is used for deployment and account factory related calls in the API: https://github.com/daimo-eth/daimo/pull/160/files/50df9851ad8bcc51b33ead873d93320af33b3948#diff-d47c5fcd14c4963fdceda8b0a2628726e1412787b1af4a426cecfd14b50bb518R282